src/transformers/tuple.ts

```diff
@@ -10,9 +11,11 @@ export class TupleTransformer {
     mapNullableType: MapNullableType,
     mapItem: MapSubSchema
   ): SchemaObject {
-    const { items } = zodSchema._def;
+    const items = zodSchema._zod.def.items;
 
-    const schemas = items.map(mapItem);
+    const schemas = items.map(item =>
+      isAnyZodType(item) ? mapItem(item) : {}
```

Comment:
Is this just due to too broad TS types or is it actually possible for this to not be a zod type?